src/org/sosy_lab/common/io/TempFile.java

```diff
@@ -30,7 +30,7 @@
 /** Utilities for temporary files. */
 public class TempFile {
 
-  private static final Path TMPDIR = Paths.get(StandardSystemProperty.JAVA_IO_TMPDIR.value());
+  public static final Path TMPDIR = Paths.get(StandardSystemProperty.JAVA_IO_TMPDIR.value());
```

Comment:
Please don't make members public except if specifically desired (and then there should be some documentation). Here, package private would be enough.